dbt/models/default/schema.yml

```diff
@@ -56,36 +63,46 @@ models:
             )
             OR
             (change is null)
+          config:
+            error_if: ">266758"
   - name: vw_card_res_char_test
     description: '{{ doc("vw_card_res_char_test") }}'
     tests:
       # Unique by card and year
       - dbt_utils.unique_combination_of_columns:
-          name: unique_by_card_and_year
+          name: vw_card_res_char_unique_by_card_and_year
           combination_of_columns:
             - year
             - card
+          config:
+            error_if: ">913"
       # char_recent_renovation correlates with char_renovation
       - dbt_utils.expression_is_true:
-          name: renovation_fields_match
+          name: vw_card_res_char_renovation_fields_match
           # char_renovation can be null, which we count here as falsey
           expression: >-
             char_recent_renovation = (
               case when char_renovation = '1' then true else false end
             )
+          config:
+            # This error threshold represents an upper bound, since the
+            # underlying query that generates `char_recent_renovation`
+            # is currently not deterministic. When we fix the underlying
+            # view to make that column deterministic, we should lower this.
```

Comment:
Very strange! I think this has to do with the ordered window functions that we use to generate `vw_card_res_char.char_recent_renovation`:

https://github.com/ccao-data/data-architecture/blob/73cdfec06aa2730d41a06558bcf469b4a3f9c124/dbt/models/default/vw_card_res_char_test.sql#L90-L124

Billy's looking into it!

Reply:
Seemingly fixed by #47.

Reply:
Yup, that fixes it -- resolving this now!